_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,310 @@
+  nodes that have been online more than a few hours---it'll wait up to 5
+  minutes before querying.  This improves the chance that a restarted
+  node will entirely use P2P address discovery without relying on the
+  centralized DNS seeds.
+
+- [LND #4228][] adds a new wallet command `labeltx` for labelling past onchain
+  transactions. This is a continuation of the work done in [LND #4213][] which
+  allowed setting a label when sending a payment.  Labels are personal
+  wallet metadata that help the user remember who they paid and what
+  they bought; the labels aren't part of the onchain transactions and
+  aren't shared with any other user.
+
+## On the occasion of Optech Newsletter #100
+
+> "I am somewhat surprised that nobody has taken to [writing] weekly
+> summaries of research and development activity. Summarizing recent
+> work is a valuable task that others can engage in just by reading the
+> mailing list and aggregating multiple thoughts together."
+>
+> {:.right}
+> ---Bryan Bishop, [19 August 2015][bishop summaries]
+
+{% comment %} <!--
+  $ find _posts/en/newsletters/ _includes/specials/bech32/  -type f | xargs cat | wc -w
+  155217
+
+  443 pages = 155217 words / 350 words per printed page
+-->{% endcomment %}
+
+Almost five years after Bishop made the above comment, we remain
+convinced that writing weekly summaries of research and development
+activity is a task that's valuable to both the open source Bitcoin
+development community and to the many businesses that depend upon the
+community's work.  But in the two years we've been producing this
+newsletter, we've also discovered that summarizing isn't quite as quick
+and simple as we initially expected it to be.  Accordingly, we'd like to
+take this chance to thank the people who make this newsletter possible
+by generously contributing a significant amount of their valuable time
+week after week: [Adam Jonas][], [Carl Dong][], [David A. Harding][],
+[John Newbery][], [Jon Atack][], [Mike Schmidt][], and [Steve Lee][].
+
+We additionally thank the experienced Bitcoin and LN contributors who
+kindly provided us with special help on certain complex topics or who
+contributed field reports and other special content to the newsletter
+over the past two years.
+
+Publishing a high-quality weekly newsletter and working to fulfill other
+aspects of Optech's [mission][about page] wouldn't be possible without
+the financial support of our member companies.  We thank them for their
+continuing commitment to improving communication between Bitcoin users,
+developers, and businesses.
+
+{:.center style="font-size: 1.5em"}
+![Thank you to our members!](/img/posts/2020-06-members.png)
+
+We also remain eternally thankful to our founding sponsors [Wences
+Casares][], [John Pfeffer][], and [Alex Morcos][] <!-- same order as on
+About page --> as well as to organizations such as [Chaincode Labs][]
+and [SquareCrypto][] who both allow and encourage their staff to use
```

Comment:
suggest omitting "both", it's ambiguous whether it refers to both orgs or both "allow and encourage", and both of the meanings are understood without it